lessons/222/elixir-app/rel/vm.args.eex

```diff
@@ -0,0 +1,10 @@
+## Customize flags given to the VM: https://www.erlang.org/doc/man/erl.html
+## -mode/-name/-sname/-setcookie are configured via env vars, do not set them here
+
+## Increase number of concurrent ports/sockets
+##+Q 65536
+
+## Tweak GC to run more often
+##-env ERL_FULLSWEEP_AFTER 10
+
++sbwt none
```

Comment:
I wish I could document Why I added `+sbwt none` since this does not come by default as part of the `mix release` and have a major impact on k8s environment.

Leaving a note here @antonputra I wish the impact of misconfiguring this one was showed case, nowadays is one of the very first things I check simply because I have seen and made the mistake myself far too many times

Reply:
> have a major impact on k8s environment

@yordis Could you add more details about the impact? Have you noticed an increase in latency?

Reply:
Check https://adoptingerlang.org/docs/production/kubernetes/#container-resources

Reply:
Also a good one: https://stressgrid.com/blog/beam_cpu_usage/